src/status_im2/contexts/syncing/scan_sync_code/style.cljs

```diff
@@ -116,7 +117,9 @@
    :justify-content   :center})
 
 (def enable-camera-access-header
-  {:color colors/white})
```

Comment:
Made the text semibold on scan screen along with onboarding screen as discussed with @pedro-et